app/pages/admin/project-status.spec.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import assert from 'assert';
+import sinon from 'sinon';
+import { shallow } from 'enzyme';
+import ProjectStatus from './project-status';
+
+const project = {
+  name: 'Test Project',
+  configuration: {
+    default_workflow: '1'
+  }
+};
+
+const workflows = [
+  {
+    active: true,
+    configuration: {},
+    display_name: 'Test Workflow 1',
+    id: '1'
+  },
+  {
+    active: true,
+    configuration: {},
+    display_name: 'Test Workflow 2',
+    id: '2'
+  },
+  {
+    active: false,
+    configuration: {},
+    display_name: 'Test Workflow 3',
+    id: '3'
+  }
+];
+
+describe('ProjectStatus', () => {
+  let wrapper;
+  let loadingIndicator;
+  let onChangeWorkflowLevelSpy;
+  let handleDialogCancelSpy;
+  let handleDialogSuccessSpy;
+  let handleToggleSpy;
+  let event;
+
+
+  before(() => {
+    onChangeWorkflowLevelSpy = sinon.stub(ProjectStatus.prototype, 'onChangeWorkflowLevel');
+    handleDialogCancelSpy = sinon.stub(ProjectStatus.prototype, 'handleDialogCancel');
+    handleDialogSuccessSpy = sinon.stub(ProjectStatus.prototype, 'handleDialogSuccess');
+    handleToggleSpy = sinon.stub(ProjectStatus.prototype, 'handleToggle');
+
+    wrapper = shallow(<ProjectStatus />);
+
+    wrapper.setState({
+      project,
+      workflows
+    });
+  });
+
+  it('renders without crashing', () => {
+    assert.equal(wrapper, wrapper);
+  });
+
+  describe('custom components', () => {
+    it('LoadingIndicator renders if project is not in state', () => {
+      wrapper.setState({ project: null });
```

Comment:
This doesn't need to happen once you remove the setState out of the before block. You do want to test what is rendered for the initial state. There should also be a unit test for when the workflows state is null (the initial state for workflows)